src/site/markdown/server/manage-webadmin.md

```diff
@@ -2816,6 +2817,29 @@ while:
  - userExportFrom: export deleted messages from this user
  - exportTo: content of deleted messages have been shared to this mail address
  - totalExportedMessages: number of deleted messages match with json query, then being shared to sharee
+ 
+### Purge Deleted Messages
+ 
+First, you need to configure 'retentionPeriod' in deletedMessageVault.
+
+Delete all expired deleted messages with 'retentionPeriod' configured.
+
+```
+curl -XPOST http://ip:port/deletedMessages?action=purge
+```
+
+Response code:
+
+ - 201: Task for purging has been created
+ - 400: Bad request: 
+   - action query param is not present
+   - action query param is not a valid action
+
+You need to create a CRON job to call this endpoint in a regular basis. Example:
```

Comment:
First, the use doesn't "need" that. You can say: "You may want to call this endpoint on a regular basis".
Please don't talk about CRON because nowadays people have a lot more solutions than CRON.